SitumWayfinding/Classes/Positioning/Search/SearchResultsTableViewController.swift

```diff
@@ -112,7 +112,6 @@ class SearchResultsTableViewController: UITableViewController {
         noDataLabel.text          = NSLocalizedString("search.noResultFound", bundle: SitumMapsLibrary.bundle, comment: "")
         noDataLabel.textAlignment = .center
         tableView.backgroundView  = noDataLabel
-        tableView.backgroundColor = UIColor.white
         tableView.separatorStyle  = .none
     }
     
```

Comment:
Estou executando a app e revisaronse as cores de SearchTableViewCell? porque a planta con estos colores oscuros case non se le. Habería que revisar a propiedade floorLabel. Ademáis os iconos que salen, que son os que salen tanem no mapa cando non están seleccionados (asumo que son os por defecto de situm) están en negro o cal fai que se vexan moi pouco..